src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -1097,6 +1097,7 @@ public async Task ReplaceClusterCert()
                                 // Exporting with the private key, but without the password
                                 certBytes = cert.Export(X509ContentType.Pkcs12);
                                 certificate.Certificate = Convert.ToBase64String(certBytes);
+                                certificate.Password = null;
```

Comment:
how about passing null in line 1127?

Reply:
What is the motivation for that?
We change the Certificate of the certificate, which means the Password is not relevant for this object (certificate).
If we decide not to touch certificate, then I think we should not touch its Certificate.
Anyway, having an object that holds a Certificate and Password that do not match is not intuitive and can cause bugs like the one in the PR's issue.

Reply:
I meant instead of setting a null on Password field here, and pass the `certiticate` around you could pass null where it's used, no?
```
var replicationTask = Server.StartCertificateReplicationAsync(newCertificate, null, replaceImmediately, GetRaftRequestIdFromQuery());
```